NuKeeper/Engine/RepositoryUpdater.cs

```diff
@@ -98,7 +100,7 @@ private void GitCloneToTempDir()
                 Console.WriteLine("Commiting");
 
                 var commitMessage = CommitReport.MakeCommitMessage(updateSet);
-                _git.Commit(commitMessage);
+                _git.Commit($":{CommitEmoji}: {commitMessage}");
```

Comment:
Ah right, it's used in 2 places. How about 2 methods, `MakeCommitMessage` which forwards most to `MakePrTitle`.